public/redux/reducers/appStateReducers.js

```diff
@@ -120,6 +120,13 @@ const appStateReducers = (state = initialState, action) => {
     };
   }
 
+  if (action.type === 'UPDATE_CLUSTER_STATUS') {
+    return {
+      ...state,
+      clusterStatus: action.clusterStatus,
```

Comment:
you are missing add to initial state the prop `clusterStatus`  for this reducer

Reply:
Thx!!! 😄